assets/sassFolder/sass.sass

```diff
@@ -0,0 +1,19 @@
+@import "variables"
+
+*
+    margin: 0
+    padding: 0
+
+@import 'background'
+/* Text edits */
+@import 'text'
+/* Top Part */
+@import 'top-part'
+/* Middle part, Avatar icon, Name,vim index.html Age, City */
+@import 'middle-img'
+/* Bottom part of image, Folowers, Likes, Photos */
+@import 'card-bottom'
+/* Small devices */
+@import 'mobile-query'
+/* Small device Landscape */
+@import 'landscape-query'
```

Comment:
Meni nije bas logicno da sass bude bas ovoliko podeljen. Zamisli na vecem projektu kakav bi to haos bio ali radi vezbe OK